gprofiler/profilers/java.py

```diff
@@ -866,7 +866,9 @@ def __init__(
             stop_event=self._profiler_state.stop_event, jattach_timeout=self._jattach_timeout
         )
         self._ap_timeout = self._duration + self._AP_EXTRA_TIMEOUT_S
-        application_identifiers.ApplicationIdentifiers.init_java(self._jattach_jcmd_runner)
+        from gprofiler.metadata.application_identifiers_java import ApplicationIdentifiersJava
```

Comment:
If we don't go with https://github.com/Granulate/gprofiler/pull/702/files#r1116174489, can you move this import to the top of the file? Maybe as `from gprofiler.metadata import application_identifiers_java` to avoid the circular import.